components/script/dom/bindings/str.rs

```diff
@@ -151,7 +151,7 @@ pub fn is_token(s: &[u8]) -> bool {
 
 /// A DOMString.
 ///
-/// This type corresponds to the [`DOMString`](idl) type in WebIDL.
+/// This type corresponds to the [`DOMString`](https://webidl.spec.whatwg.org/#idl-DOMString) type in WebIDL.
 ///
 /// [idl]: https://heycam.github.io/webidl/#idl-DOMString
```

Comment:
Now the separate "idl" link is unused. Does this fix the issue as well?

```suggestion
/// This type corresponds to the [`DOMString`][idl] type in WebIDL.
///
/// [idl]: https://webidl.spec.whatwg.org/#idl-DOMString
```

Reply:
Since the intention is to reference an external doc, would implementing either of the following two options be acceptable?

1. Following the [`Easy`] pattern in [the rust doc](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#getting-started).

```
/// This type corresponds to the [`DOMString`] type in WebIDL.
///
/// [`DOMString`]: https://webidl.spec.whatwg.org/#idl-DOMString
///
/// Conceptually, a DOMString has the same value space as a JavaScript String,
```

2. Deleting the separate "idl" link:

```diff
/// This type corresponds to the [`DOMString`](https://webidl.spec.whatwg.org/#idl-DOMString) type in WebIDL.
///
-/// [idl]: https://heycam.github.io/webidl/#idl-DOMString
/// 
/// Conceptually, a DOMString has the same value space as a JavaScript String,
```

Reply:
Yes, the first one looks good to me as long as the link still works. Thank you.